snipssonos/snipssonos.py

```diff
@@ -21,34 +18,36 @@
 class SnipsSonos:
     """ Sonos skill for Snips. """
 
-    def __init__(self, spotify_refresh_token=None, speaker_index=None, locale=None):
-        # find the device
-        devices = soco.discover()
-        if devices is None or len(list(devices)) == 0:
-            time.sleep(1)
+    def __init__(self, spotify_refresh_token=None, speaker_index=None, locale=None, sonos_ip=None):
```

Comment:
Maybe explicitly show that the skill does not work on multiple sonos. (in readme or change naming)